README.md

```diff
@@ -16,12 +16,12 @@
 Easily create a new Python project with simple commands:
 
 ```shell
-pipx run acpp -n my-project
+pipx run ansys-create-python-project -n my-project
 ```
 or
 
 ```shell
-python -m acpp -n my-project
+acpp -n my-project
```

Comment:
Does this command require to preinstall the package and/or set any thing in PATH or PYTHONPATH ?

Reply:
you need to install using pip once, and then you can use the command.